### PR TITLE
Adding Save & Reload testcase for ACL

### PR DESCRIPTION
**Pull request by Anandaraj-Maharajan**
This checkin adds changes to perform save and reload test case for ACL rules

### Description of PR

Fixes # (issue)

### Type of change
- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [X] Test case(new/improvement)

### Approach
How did you do it?
Installed the ACL rules in the switch, 
pushed the changes to configDB, 
performed a reload, 
re-run the traffic test
How did you verify/test it?
Using PTF traffic test, verified the desired packets are blocked after save and reload. 
Any platform specific information?
Supported testbed topology if it's a new test case?
[t1, t1-lag, t1-64-lag]
[aclreload.log](https://github.com/Azure/sonic-mgmt/files/1769509/aclreload.log)

### Documentation 

successful test run log in T1 topology is attached along with pull request.

### CODE REVIEW

File: ansible/roles/test/tasks/acltb.yml [hunk 1]
```diff
@@ -115,17 +115,8 @@
...
-      copy: src="roles/test/tasks/acl/{{ item }}" dest="/tmp/"
-      with_items:
-        - "acltb_test_rules-del.json"
-      
     - name: Clean up ACL rules.
-      vars:
-        command_to_run: "acl-loader update full /tmp/acltb_test_rules-del.json"
-        errors_expected: false
-      include: roles/test/tasks/run_command_with_log_analyzer.yml
+      command: acl-loader delete
```
Comment (prsunny) [inline ansible/roles/test/tasks/acltb.yml]: Why this change as compared to the previous one?. This will delete the entire set of rules but the previous approach deletes the rules based on the json file.

Comment (Anandaraj-Maharajan) [reply ansible/roles/test/tasks/acltb.yml]: @prsunny  acltb_test_rules-del.json file is a empty file. So using either json or 'delete' command performs the same action. Using acl-loader delete is straight forward and reduces the number of lines of code, so made this change. 

Additionally I verified acl-loader delete command doesn't remove the DATAACL applied on the interfaces.
